### PR TITLE
Technical debt seminar (en-02)

### PR DESCRIPTION
**Pull request by rosagarlo**

### CODE REVIEW

File: slides/course2122/seminars/DocEn-02 [hunk 1]
```diff
@@ -0,0 +1 @@
+
```
Comment (labra) [inline slides/course2122/seminars/DocEn-02:1]: Is this file necessary?

**Review by labra — COMMENTED** [review]
Could you also change the html file adding links to the slides and the report?

**Comment by labra** [comment]
> Could you also change the html file adding links to the slides and the report?

Forget this comment, I have seen that you did it in the other PR

File: slides/course2122/seminars/DocEn-02 (hunk 1 above)
Comment (rosagarlo) [reply slides/course2122/seminars/DocEn-02:1]: No but I couldn't delete it, sorry for the confusion
